runtime/doc/builtin.txt

```diff
@@ -10114,6 +10114,20 @@ visualmode([{expr}])						*visualmode()*
 		a non-empty String, then the Visual mode will be cleared and
 		the old value is returned.  See |non-zero-arg|.
 
+visualtext()						*visualtext()*
```

Comment:
Minor nitpicking: For adding new tags, I don't know whose responsibility it is to add to the [tags](https://github.com/vim/vim/blob/master/runtime/doc/tags) file. Normally I would assume Bram just does it himself when doing runtime file updates but even then the tags are not always updated. Can this PR just add it if it's going to be a direct merge? You can either run the tags generation or what I sometimes do is to just manually add it to the tags file (need to make sure format is correct and sorted).

Reply:
Oh actually now I remember it I think we added a "vimtags" check to the [ci.yml ](https://github.com/vim/vim/blob/master/.github/workflows/ci.yml) file so it would check in CI. So we should definitely add the tags to pass CI.